base_cancel_confirm/model/base_cancel_confirm.py

```diff
@@ -25,6 +27,19 @@ class BaseCancelConfirm(models.AbstractModel):
         help="An optional cancel reason",
     )
 
+    def _cancel_confirm_disabled(self):
+        key = "%s.cancel_confirm_disable" % self._name
+        res = self.env["ir.config_parameter"].get_param(key)
+        if not res:
+            return False
```

Comment:
@kittiu What I wanted to say is this is the default if no it config parameter entry.

Should be true here

Reply:
You are genius ! :)

Reply:
> You are genius ! :)

Don't over estimate me ;-)

Reply:
I think you are rock star now. Very active contributor 🥇